h/tasks/annotations.py

```diff
@@ -0,0 +1,38 @@
+from sqlalchemy import func, update
+
+from h.celery import celery, get_task_logger
+from h.models import Annotation, User
+
+log = get_task_logger(__name__)
+
+
+@celery.task
```

Comment:
Keep in mind that h's global Celery config has a [hard time limit of `240` secs](https://github.com/hypothesis/h/blob/07b22e03b906e50c607086163799cee11037ad5f/h/celery.py#L75) (4 mins) for task runs. If you're going to use h-periodic to run this task periodically and you only want one instance of the task to be running at a time then I think h-periodic must not run the task more than once every 4 mins.

Alternatively if this task is expected to complete much faster than 4 mins then you can add a shorter `soft_time_limit` and `time_limit` here in the `@celery.task` decorator and then h-periodic will be free to run the task more frequently.

Reply:
To be honest I don't know what to expect 30s? 1m? 5m?

will adjust after the first few runs.